indico/web/client/js/react/components/notes/NoteEditor.jsx

```diff
@@ -14,35 +16,91 @@ import {handleSubmitError} from 'indico/react/forms';
 import {FinalModalForm} from 'indico/react/forms/final-form';
 import {Translate} from 'indico/react/i18n';
 import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
+import {camelizeKeys} from 'indico/utils/case';
 
+import {FinalMarkdownEditor} from '../MarkdownEditor';
 import {FinalTextEditor} from '../TextEditor';
 
+// import './NoteEditor.module.scss';
```

Comment:
what was the problem with this?

Reply:
Oops, nothing, but that was a remnant of the time where I had the full Markdown Editor in that file. It's now `MarkdownEditor.module.scss` since it's all styling for that one!